provisioning/device/src/RegisterInput.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Shared;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client
+{
```

Comment:
I don't follow this class. What is the usecase?

Reply:
We do not need Register input here or anywhere. I'll remove it